assets/js/youcan-pay.js

```diff
@@ -38,17 +43,25 @@ jQuery(function ($) {
                     }
                 }, 500);
 
+                //TODO: need to manage this step by selected gateway
+                console.log('window.ycPay.selectedGateway: ' + window.ycPay.selectedGateway);
+
                 window.ycPay.pay(data.token_transaction)
                     .then(function (transactionId) {
                         detach_loader($form, loader);
 
-                        if (typeof (data.redirect_url) !== 'undefined') {
-                            window.location.href = data.redirect_url;
-                            return;
-                        }
+                        console.log("i'm here 1!");
+
                         if (typeof (data.redirect) !== 'undefined') {
+                            console.log('i\'m here 2!');
+                            console.log('transaction_id', transactionId);
```

Comment:
those alsoe

Reply:
https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/26/commits/e7ee34f8a6f31459dd81379f445f313c66ef2877